index.bs

```diff
@@ -9853,6 +9853,13 @@ Every {{AudioWorkletProcessor}} has an associated <dfn>active source</dfn> flag,
 the node to be retained in memory and perform audio processing in
 the absence of any connected inputs.
 
+Once the construction of the node is completed,
+<dfn>processor construction data</dfn> will be created and
+trasnferred to the matching {{AudioWorkletProcessor}}'s
+{{AudioWorkletProcessor()|constructor}}. This data contains
```

Comment:
Maybe "The `<a>processor construction data</a> contains...`

Reply:
Hmm.  This is ill-defined, because I can't tell what the data actually is and there's no link to what it might be.  But later on, in the AudioWorkletNode constructor algorithm, step 12, you say what the processor construction data actually is.  Thus, step 12 has actual definition of what the processor contruction data is.  Not sure if this is what you want, but this at least tells you what the data really is, instead of some abstract concept of data here.

Reply:
That was the original form, but the problem here is we can't define a thing in the middle of an algorithm. (a bikeshed rule) That's why the definition has to be moved to here.